### ISSUE TITLE
Missing method

### ISSUE THREAD

**Issue by bjaxelsen** [agentrickard/domain#64]
domain_config/src/DomainConfigOverrider.php needs to implement getCacheableMetadata - I am not quite sure how to implement this properly, but I created this dummy method:

```
--- a/domain_config/src/DomainConfigOverrider.php
+++ b/domain_config/src/DomainConfigOverrider.php
@@ -12,6 +12,7 @@ use Drupal\Core\Config\StorageInterface;
 use Drupal\Core\Config\ConfigFactoryOverrideInterface;
 use Drupal\domain\DomainNegotiatorInterface;
 use Drupal\domain\DomainInterface;
+use Drupal\Core\Cache\CacheableMetadata;

 /**
  * Domain-specific config overrides.
@@ -117,5 +118,13 @@ class DomainConfigOverrider implements ConfigFactoryOverrid
     return NULL;
   }

+  /**
+   * {@inheritdoc}
+   * Fixed
+   */
+  public function getCacheableMetadata($name) {
+    $metadata = new CacheableMetadata();
+    return $metadata;
+  }
 }
```

**Comment by bjaxelsen** [comment]
(running D8 beta 14)

**Comment by agentrickard** [comment]
This is probably part of the much larger issue #33.

**Comment by agentrickard** [comment]
This is fixed in #33, but the module tests still fail.
